http-ui/Cargo.toml

```diff
@@ -37,3 +37,6 @@ fst = "0.4.5"
 
 # Temporary fix for bitvec, remove once fixed. (https://github.com/bitvecto-rs/bitvec/issues/105)
 funty = "=1.1"
+
+[dev-dependencies]
+serde_test = "1.0.125"
```

Comment:
Could you please set your editor to always add the final newline to the edited files, please?

Reply:
Yeah, sorry for that 😬  Just the CLion's settings to do so, thanks